<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BusMonitor</class>
 <widget class="QMainWindow" name="BusMonitor">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>707</width>
    <height>335</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Bus Monitor</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../data/qModbusMonitor.qrc">
    <normaloff>:/img/view-16.png</normaloff>:/img/view-16.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QListView" name="lstRawData"/>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionClear">
   <property name="icon">
    <iconset resource="../data/qModbusMonitor.qrc">
     <normaloff>:/img/pause-16.png</normaloff>:/img/pause-16.png</iconset>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
   <property name="toolTip">
    <string>Clear</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="../data/qModbusMonitor.qrc">
     <normaloff>:/img/close16.png</normaloff>:/img/close16.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="toolTip">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="../data/qModbusMonitor.qrc">
     <normaloff>:/img/save-16.png</normaloff>:/img/save-16.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="toolTip">
    <string>Save</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../data/qModbusMonitor.qrc"/>
 </resources>
 <connections/>
</ui>
